docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md

```diff
@@ -19,7 +19,7 @@ The TargetFrameworkName was previously null in the default app domain, unless it
 Code should be updated to not depend on `AppDomainSetup.TargetFrameworkName` defaulting to null. If it is required that this property continue to evaluate to null, it can be explicitly set to that value.
 
 ### Affected APIs
-* `M:System.AppDomainSetup.get_TargetFrameworkName`
+* `P:System.AppDomainSetup.TargetFrameworkName`
```

Comment:
This change does specifically affect the getter (and not setter), so should it maybe be left with the more narrow accessor doc id?